tests/kas.spec.ts

```diff
@@ -91,8 +91,10 @@ test('test Kafka list filtered by status', async ({ page }) => {
 
   await deleteKafkaInstance(page, testInstanceName, false);
 
-  await filterByStatus(page, 'Deleting');
-  await expect(page.getByText(testInstanceName)).toBeTruthy();
+  // TODO: Probably race condition when deleting is finished too quickly
```

Comment:
Can you please revert this change and discuss it in a separate PR/Issue?

I agree that this test is going to be unreliable and also the next one should be re-evaluated:
https://github.com/redhat-developer/consoledot-e2e/blob/85a3faf7bee007726010bd83bf4db0c3f7d43698/tests/kas.spec.ts#L106

But, at the moment, this is running "reliably enough" in CI ...

Reply:
ok, I will keep only the comment as a new change